### ISSUE TITLE
Try to save new entry without any required fields - form is not scrolled to top, so user can't see validation message

### ISSUE THREAD

**Issue by antos-k** [diglabby/mapa#97]
1. Open map
2. Press 'Add new entry' button
3. Leave any required field empty
4. Go to the bottom of item creation form
5. Press 'Save'

**Result:** Form is not scrolled to top, so we can't see 'Something is missing, please check below.' message - user can to not understand, why nothing happens when he press Save.
**Expected result:** Form is scrolled to the first validation message (so, user can see it and correct the reason)

**Comment by wellemut** [comment]
The Team in Munic is doing a big refactoring till end of September. this bug should be solved afterwards, because it might be solved during refactoring.

**Comment by rizomaa** [comment]
@antos-k could you show a screenshot for this bug?

**Comment by PolinaSam** [comment]
this bug hasn't been solved yet.

**Comment by k-gorod** [comment]
I cant imagine how to work with that naming methodology. 
![image](https://user-images.githubusercontent.com/58625453/82731052-d5c75400-9d0c-11ea-86bf-b3a103662746.png)
Would you like to tell me, how can i work with elements?
But for now, i did that:
<NavButton
            keyName = "save"
            classname = "pure-u-1-2"
            onClick = { () => {
              this.props.handleSubmit();
              document.getElementsByClassName('pure-form')[0].scrollIntoView({block: "start", behavior: "smooth"});
            }}
            icon = "save"
            text = { t("save") }
          />

https://youtu.be/hZ4JUPD3Aoo
Is that action what we looking for?

**Comment by rizomaa** [comment]
@k-gorod yes, you are in a right direction please make a pull request to understand your whole 
solution.

**Comment by k-gorod** [comment]
Im noticed that some of elements have names. I used it to check them values. PR created

**Comment by k-gorod** [comment]
PR:
https://github.com/diglabby/mapa/pull/349

**Comment by rizomaa** [comment]
@k-gorod дзякуй за ПР, у бліжэйшы час праглядзім

**Comment by rizomaa** [comment]
былі думы наконт таго, ці ёсць якая лібы для больш прыгожага рашэння, але акей, архітэктурна нейтральна там усё прымаю. будзем тэставаць)
